Makefile

```diff
@@ -1,5 +1,8 @@
 MAKEFLAGS += --no-builtin-rules
 
+SHELL = /bin/bash
+.SHELLFLAGS = -o pipefail -c
+
```

Comment:
This is so that `$(CPP) $(CPPFLAGS) -I include $< | $(AS) $(ASFLAGS) -o $@` will cause the build to fail if `cpp` errors (which happens for cases such as not being able to find an include)

Reply:
Could you add a comment near those lines to explain just that?

Reply:
I've added a brief comment that's hopefully enough to clarify this